### PR TITLE
Update review-set-search.md

### PR DESCRIPTION
**Pull request by ninacchen**
Added a "you" as the sentence sounds strange without a subject. Please modify as fit.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/review-set-search.md [hunk 1]
```diff
@@ -59,6 +59,6 @@ In addition to queries that you can save, you can use review set filters to quic
 
 Filters differ from queries in two significant ways:
 
-- Filters are transient. They don't persist beyond the existing session. In other words, you can't save a filter. Queries are saved to the review set, and access them whenever open the review set.
+- Filters are transient. They don't persist beyond the existing session. In other words, you can't save a filter. Queries are saved to the review set, and access them whenever you open the review set.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/review-set-search.md:62]:
```suggestion
- Filters are transient. They don't persist beyond the existing session. In other words, you can't save a filter. Queries are saved to the review set, and you can access them whenever you open the review set.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by markjjo** [comment]
@ninacchen PR merged. Thanks for submitting this. Tip for future PRs. If you @ comment me in the comment then I will get a notification about the PR. It streamlines the workflow especially for quick fixes like this one.

**Pull request merged**